dist/static/site.css

```diff
@@ -0,0 +1,366 @@
+html,
+body {
+  width: 100%;
+  height: 100%;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: "Source Sans Pro", sans-serif;
+  color: #fff;
+}
+
+header {
+  text-align: center;
+}
+
+#heading {
+  font-family: "Fascinate Inline", monospace;
+  font-size: 3em;
+  user-select: none;
+  margin: 0;
+  padding: 30px;
+}
+
+h1,
+h2 {
+  text-shadow: 2px 2px 4px #000000;
+}
+
+h2 {
+  font-weight: 600;
+}
+
+@keyframes pop-in {
+  from {
+    transform: translateY(-20px);
+    opacity: 0;
+  }
+  to {
+    transform: translateY(0px);
+    opacity: 1;
+  }
+}
+
+#nameentry {
+  position: absolute;
+  top: 20vh;
+  left: 20vw;
+  background: #111;
+  padding: 15px 30px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  width: 60vw;
+  animation: 250ms pop-in ease;
+}
+
+button,
+input {
+  font-family: "Source Sans Pro", sans-serif;
+}
+
+input[type="text"] {
+  font-family: monospace;
+  background: #1e1e1e;
+  color: #fff;
+  border-radius: 5px;
+  border: 2px solid transparent;
+  padding: 5px 8px;
+  outline: none;
+  box-sizing: border-box;
+  transition: 40ms ease-in-out border-color;
+}
+
+input:focus {
+  background: #222;
+  border: 2px solid #a50bca;
+}
+
+#nameentry input {
+  font-size: 28px;
+  width: 100%;
+  padding: 10px 15px;
+  margin: 10px 0;
+}
+
+button {
```

Comment:
Same for buttons too, adding styling for disabled state. Maybe gray + disabled cursor?